data/spells/scripts/support/food.lua

```diff
@@ -1,19 +1,19 @@
-local food = {
+local foods = {
 	2666, -- meat
 	2671, -- ham
 	2681, -- grape
 	2674, -- apple
 	2689, -- bread
 	2690, -- roll
-	2696  -- cheese
+	2696 -- cheese
 }
 
 function onCastSpell(creature, variant)
-	creature:addItem(food[math.random(#food)])
-	if math.random(1, 100) > 50 then
-		creature:addItem(food[math.random(#food)])
+	if math.random(1, 100) >= 50 then
```

Comment:
Not exactly, ``math.random(1, 100)`` is a closed interval [a, b], so that [1, 100]  (has a ``b - 1 + 1`` which is ``100 - 1 + 1`` that is 100 different numbers (including 100), so ``>=`` could be read in a reversed order: ``50 <= math.random(1, 100)``, so if the random value was **50** we would get ``50 <= 50`` (b - a + 1) which is 1/2.
If I used ``<`` it would be a open interval (b - a), so it would exclude one element.

I had done it wrong when I updated this script a while ago, so I have now corrected the change.
The code is clearer with that statement moved below the ``if`` I guess.